protostar-rust/src/pretty_printing.rs

```diff
@@ -43,9 +45,10 @@ pub fn print_test_result(test_name: &str, result_value: &RunResultValue) {
 
     if result_str.is_empty() {
         println!("[{result_tag}] {test_name}");
+    } else if passed {
+        println!("[{result_tag}] {test_name}\n\nSuccess data:{result_str_bold}\n");
```

Comment:
Is it possible to have success data at all?

Reply:
Not sure, but the compiler seems to allow it
```rust
/// The ran function return value.
#[derive(Debug, Eq, PartialEq)]
pub enum RunResultValue {
    /// Run ended successfully, returning the memory of the non-implicit returns.
    Success(Vec<Felt252>),
    /// Run panicked, returning the carried error data.
    Panic(Vec<Felt252>),
}
```

Reply:
Okay, better to explicitly handle it then. We can consider creating an issue in triage for investigating this further.

Reply:
cc @MaksymilianDemitraszek

Reply:
It is determined based on first felt if 0 then it is success otherwise panic,